tests/test_goniometer.py

```diff
@@ -206,7 +205,7 @@ def test_goniometer_from_phil():
         parse(
             """
     goniometer {
-      axes = (1, 0, 0)
+      axis = (1, 0, 0)
```

Comment:
It would probably be useful to keep a test that setting a single axis via the `axes=` parameter does actually work (if we want to continue supporting this behaviour).

Reply:
Done in f789a56bfaa6debe60ef584a9961405e9d44b256 but I forgot that from the commit message